src/utils/config.py

```diff
@@ -14,6 +15,26 @@
 _CONFIG_SINGLETON = None
 
 
+def get_sample_service_url(sw_url, ss_release):
+    """"""
+    payload = {
+        "method": "ServiceWizard.get_service_status",
+        "id": '',
+        "params": [{"module_name": "SampleService", "version": ss_release}],
+        "version": "1.1"
+    }
+    headers = {'Content-Type': 'application/json'}
+
+    sw_resp = requests.post(url=sw_url, headers=headers, data=json.dumps(payload))
+    if not sw_resp.ok:
+        raise RuntimeError(f"ServiceWizard error, with code {sw_resp.status_code}")
```

Comment:
One more little thing, can you throw in `respt.text` into the error message.
Getting the response text will be useful when looking at the logs.